ckan/lib/plugins.py

```diff
@@ -524,3 +524,31 @@ def activity_template(self):
         return 'organization/activity_stream.html'
 
 _default_organization_plugin = DefaultOrganizationForm()
+
+
+class DefaultTranslation(object):
+    def directory(self):
+        '''Change the directory of the *.mo translation files
+
+        The default implementation assumes the plugin is
+        ckanext/myplugin/plugin.py and the translations are stored in
+        i18n/
+        '''
+        import os
+        return os.path.abspath(os.path.join(os.path.dirname(__file__),
```

Comment:
yeah it unfortunately does, but this is just the default implementation that most people will have if they ran a `paster --plugin=ckan create`, but to the people who move their plugin to a different location they will just have to tolerate overriding `directory()` with this implementation.